frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -131,13 +155,21 @@ const DisksListPage: React.FC<{ obj: NodeKind }> = (props) => {
           selector: { [LABEL_SELECTOR]: nodeName },
         },
       ]}
+      customData={{
+        diskOsdMap: new Map() /* TBD(Afreen) Will be changed to actual state with this https://issues.redhat.com/browse/RHSTOR-1194  */,
```

Comment:
A question - where are you exactly adding values inside `diskOsdMap` as you are getting the disk name from this Map?

Reply:
I will add the local state to disk list page which will poll data and the pass it down the children.